draft-ietf-quic-multipath.md

```diff
@@ -877,15 +695,15 @@ the server provided N Connection IDs, and the client is already actively
 using N paths, the limit is reached. If the client wants to start
 a new path, it has to retire one of the established paths.
 
-ACK_MP frame (defined in {{ack-mp-frame}}) can be returned via either a
-different path, or the same path identified by the Path Identifier, based on
-different strategies of sending ACK_MP frames.
+ACK_MP frame (defined in {{ack-mp-frame}}) SHOULD be send on the same path
+as identified by the Path Identifier. However, an ACK_MP frame can be returned via a
```

Comment:
No strong opinion here, but should the "an ACK_MP frame can be returned" be instead "an ACK_MP frame MAY be returned"?

Reply:
No, the normative part is the active part ("SHOULD send"). This other sentence should not be normative because it's just explaining that this can happen and needs to be considered.